src/ops.rs

```diff
@@ -114,18 +119,42 @@ impl Sodg {
     /// g.add(42).unwrap();
     /// let data = Hex::from_str_bytes("hello, world!");
     /// g.put(42, data.clone()).unwrap();
-    /// assert_eq!(data, g.data(42).unwrap());
+    /// assert_eq!(data, g.peek_data(42).unwrap());
     /// ```
     ///
     /// If vertex `v1` is absent, an `Err` will be returned.
-    pub fn data(&self, v: u32) -> Result<Hex> {
+    pub fn peek_data(&self, v: u32) -> Result<Hex> {
```

Comment:
@UARTman there should be no difference between "peeking" data and taking data. We should have only one function `data` to retrieve the data. What happens behind the scene must not be visible to the end user.